soh/soh/OTRGlobals.h

```diff
@@ -21,7 +21,6 @@ class OTRGlobals
     std::shared_ptr<Ship::Window> context;
     std::shared_ptr<SaveStateMgr> gSaveStateMgr;
     std::shared_ptr<Randomizer> gRandomizer;
-    uint16_t getItemModIndex;
```

Comment:
your main PR comment didn't explain this change. does removing this fix something or just clean stuff up?

Reply:
I removed it along with the unneeded references to it thinking it might fix the bug (I don't have a way to test this on the consoles that had issues yet). It didn't end up fixing the bug but it did need to be cleaned up so I opted not to revert.